src/components/Styles.js

```diff
@@ -26,6 +27,17 @@ export const overlayStyle = {
   opacity: 0.75,
 };
 
+export const containerStyle = {
```

Comment:
Rename to BaseStyles.js and move out any style that is only used for a specific component